eks/cdk/eks.py

```diff
@@ -75,14 +77,38 @@ def __init__(self, app: core.App, id: str, props, **kwargs) -> None:
             }
         )
 
+        eks.add_helm_chart("chaos-mesh",
+            chart = "chaos-mesh",
+            release = "chaos-mesh",
+            version = "2.2.2",
+            repository = "https://charts.chaos-mesh.org",
+            namespace = "chaos-mesh",
+            create_namespace = True
+        )
+
+        # fis role
+        fis_role = aws_iam.Role(self, "fis-role", assumed_by = aws_iam.ServicePrincipal("fis.amazonaws.com"))
+        fis_role.add_managed_policy(aws_iam.ManagedPolicy.from_aws_managed_policy_name("AdministratorAccess"))
+
+        with open('../kubernetes/manifest/cm-manager.yaml', 'r') as f:
+            cmm_manifest = yaml.safe_load(f)
+
+        eks.add_manifest("cm-manager", cmm_manifest)
+        f.close()
+
+        # update aws-auth config map
+        eks.aws_auth.add_role_mapping(fis_role, groups=["system:masters", "chaos-mesh-manager-role"])
```

Comment:
This is code to create a kuberentes configmap in the kube-system namespace for IAM roles and Kuberentes RBAC mappings.